settings.yaml

```diff
@@ -0,0 +1,129 @@
+HighFrequencyTradingService:
```

Comment:
I think it's a great time to refactor our settings section. Remove duplicates, obsolete settings; regroup them; use shared variables.

Reply:
I already removed all unused and obsolete settings